src/main/java/io/github/christophermanahan/carnitas/HTTPResponse.java

```diff
@@ -4,9 +4,10 @@
 
     static final String VERSION = "HTTP/1.1";
     static final String GET = "200 OK";
+    static final String CL = "Content-Length: 0";
     static final String CRLF = "\r\n";
 
     public byte[] serialize() {
-        return String.format("%s %s%s%s", VERSION, GET, CRLF, CRLF).getBytes();
+        return String.format("%s %s%s%s%s%s", VERSION, GET, CRLF, CL, CRLF, CRLF).getBytes();
```

Comment:
I would pull `%s %s%s%s%s%s` into it's own constant since there's a lot being passed into `String.format` it can get hard to read.